docs/catalog/cloud/containerize-your-workload-where-applicable.md

```diff
@@ -0,0 +1,36 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- compute
+- kubernetes
+- role:cloud-engineer
+- size:medium
+---
+
+# Containerize your workload where applicable
```

Comment:
simplify to `Containerize your workloads` and move the applicability discussion to the solution and/or considerations and assumptions sections below.